website/content/docs/api-gateway/install.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: Install Consul API Gateway
-description:  >-
-  Installing Consul API Gateway
+description: >-
+  Learn how to install Custom Resource Definitions (CRDs) and configure the Helm chart so that you can run Consul API Gateway on your Kubernetes deployment.
```

Comment:
```suggestion
  Learn how to install custom resource definitions (CRDs) and configure the Helm chart so that you can run Consul API Gateway on your Kubernetes deployment.
```
These are regular nouns that we've gotten into the habit of sometimes capitalizing.